### PR TITLE
Log in page visuals

### PR DESCRIPTION
**Pull request by ESteinman**
### User story
```
As a user
In order to access my existing account
I would like to see a login button that takes me to a login page,
with a login form and I would like to be able to fill in the login form and click login
```
### What we have learned
Working with tables for Scenarios and tests
Working with binding-pry debugger
"Then show me the page"-command for testing
_paths in Capybara

### CODE REVIEW

File: features/log_in_page.feature [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Feature: User wants to login
+As a user
+In order to access my existing account
+I would like to see a login button that takes me to a login page,
+with a login form and I would like to be able to fill in the login form and click login
+
```
Comment (faraznaeem) [inline features/log_in_page.feature]: Please use proper indentation

File: features/log_in_page.feature [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Background: 
+Given the following user exist in the database
+|email|name|password|password_confirmation|
+|random@test.com|randomguy|my-password|my-password|
+
```
Comment (faraznaeem) [inline features/log_in_page.feature]: Please format the background properly for easier readability

File: features/log_in_page.feature [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    Scenario: Log in page view
+    When I visit the site
+    When I click on 'Login'
```
Comment (faraznaeem) [inline features/log_in_page.feature]: Two `When` statements seems wrong, please refactor this and use proper indentation

File: features/step_definitions/basic_steps.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    end 
+  end
+  
+  Then("I fill in {string} with {string}") do |field, content|
+    fill_in field, with: content
+    end
+  
+  Then("I should see {string}") do |expected_content|
+    expect(page).to have_content expected_content
+    end
```
Comment (faraznaeem) [inline features/step_definitions/basic_steps.rb]: Please format the code and use proper indentation

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: features/log_in_page.feature (hunk 1 above)
Comment (SophieSjoberg) [reply features/log_in_page.feature]: This User story seems unnecessary long.. How about:
"In order to access my account,
I would like to be able to log in" ??

File: features/log_in_page.feature [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    Then I'm sent to the "Login" page
```
Comment (SophieSjoberg) [inline features/log_in_page.feature]: You have to assert that you will be on a specific page. "Then I should be on "Login" page"

**Review by SophieSjoberg — CHANGES_REQUESTED** [review]

**Review by faraznaeem — APPROVED** [review]

**Review by SophieSjoberg — APPROVED** [review]

**Pull request merged**